src/transform/EditTransform.js

```diff
@@ -64,11 +71,63 @@ const newEditSectionHeader = (document, index, level, titleHTML, showEditPencil
     element.appendChild(button)
   }
 
+  if (anchor && anchor.length > 0) {
+    // TODO: consider renaming this 'id' to 'anchor' for clarity - would need to update native
+    // code as well - used when TOC sections made to jump to sections.
+    element.id = anchor
+  }
+
   return element
 }
 
+/**
+ * Lead section header is a special case as it needs to show article title and description too,
```

Comment:
This code should also work for any kinds of pages, but just article pages. Would you mind changing here wherever it says article to page?

Reply:
Will do!

Reply:
Fixed!